### PR TITLE
Case 22361: openUrl fix (82)

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/22361

Test plan: https://github.com/highfidelity/hifi/pull/15456

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18980/HighFidelity-Beta-PR15458-5f3a37d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18980/HighFidelity-Quest-Beta-PR15458-5f3a37d.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18980/HighFidelity-Beta-PR15458-5f3a37d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18980/HighFidelity-Beta-Interface-PR15458-5f3a37d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18980/HighFidelity-Beta-PR15458-5f3a37d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18980/HighFidelity-Beta-Interface-PR15458-5f3a37d.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18987/HighFidelity-Beta-PR15458-b9ba764.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18987/HighFidelity-Beta-Interface-PR15458-b9ba764.dmg

File: libraries/ui/src/FileDialogHelper.cpp [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #include <QtCore/QRegularExpression>
 #include <QDesktopServices>
 
+std::function<void(const QString&)> FileDialogHelper::_openDiretoryOperator = nullptr;
```
Comment (AndrewMeadows) [inline libraries/ui/src/FileDialogHelper.cpp]: Typo: `Diretory` --> `Directory`.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18987/HighFidelity-Beta-PR15458-b9ba764.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18987/HighFidelity-Beta-Interface-PR15458-b9ba764.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18987/HighFidelity-Beta-PR15458-b9ba764.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18987/HighFidelity-Quest-Beta-PR15458-b9ba764.apk)

File: libraries/ui/src/FileDialogHelper.cpp (hunk 1 above)
Comment (SamGondelman) [reply libraries/ui/src/FileDialogHelper.cpp]: will fix in 83 PR

**Pull request merged**